.gitignore

```diff
@@ -108,3 +108,6 @@ dist
 
 # TernJS port file
 .tern-port
+
+# scss
+core/client/_theme.scss
```

Comment:
The problem with this solution is that the project crashes on the import because, by default, the 
`_theme.scss` file doesn't exist, I'm looking for a solution with an optional import system

Reply:
Yes, don't worry. @ramnes suggested me to use something like this `document.querySelector(':root').style.setProperty('--main-color', Meteor.settings.public.mainColor)` to update it in the first runtime, with this, we won't need to have a separate file.

I will update my PR soon

Reply:
Oh yes, it's a good idea!
I looked on my side in SCSS and it's complicated, there are optional imports but it only exists in LESS 😕

Reply:
Ok, looks like it's more complicated than I thought.. we can't combine `var(--primary-color)` with the `darken()` function of SCSS https://stackoverflow.com/questions/64595291/is-there-a-way-to-use-sass-lighten-and-darken-with-css-variables

The only way to use CSS variables, would be to use a similar function like this https://stackoverflow.com/a/55330103 ... And actually we use darken() many times. Same as ligther().